rest-api-spec/build.gradle

```diff
@@ -39,50 +39,133 @@ dependencies {
   clusterModules project(':modules:analysis-common')
   clusterModules project(':modules:health-shards-availability')
   clusterModules project(':modules:data-streams')
+  clusterModules project(':modules:ent-search')
```

Comment:
that should be `:x-pack:plugin:ent-search`

Reply:
```suggestion
  clusterModules project(':x-pack:plugin:ent-search')
```